js/src/donations/PaymentForm.js

```diff
@@ -158,11 +158,21 @@ export class PaymentForm_ extends React.Component {
           <Row className="justify-content-center">
             <Col lg="8">
               {this.props.config.long_description ? <hr/> : null}
-              <Input field={gift_aid_field} value={this.state.gift_aid}
-                     disabled={this.state.submitting}
-                     onChange={gift_aid => this.setState({gift_aid})}/>
 
-              <Collapse isOpen={this.state.gift_aid}>
+              <StripeForm submitting={this.state.submitting}
+                          payment_state={this.state.payment}
+                          setPaymentState={this.setPaymentState}/>
+
+              <hr/>
+
+              <strong>
```

Comment:
do you need `strong` here as well as `b` above?

Reply:
Bootstrap adds an element with a 'text-muted' class around the label text. Another css class would be needed to override and this was cleaner.